### PR TITLE
revert product-os/flowzone/pull/953

### PR DESCRIPTION
**Pull request by ab77**
* installing Python dependencies on runner [breaks](https://github.com/balena-io/environment-playground/actions/runs/8425936303/job/23073177694?pr=62#step:13:38) other dependencies (e.g. AWS-CLI)

change-type: patch

### CODE REVIEW

**Comment by klutchell** [comment]
I prefer [this](https://github.com/product-os/flowzone/pull/953#issuecomment-2018787249) approach rather than hardcoding runner tags.

**Comment by ab77** [comment]
> I prefer [this](https://github.com/product-os/flowzone/pull/953#issuecomment-2018787249) approach rather than hardcoding runner tags.

How is this approach fix the underlying condition that the setup-python action will **only** run on X64/Linux currently?

File: flowzone.yml [hunk 1]
```diff
@@ -3184,7 +3184,8 @@ jobs:
 
   cloudformation_test:
     name: Test CloudFormation
-    runs-on: ${{ fromJSON(inputs.cloudformation_runs_on || inputs.runs_on) }}
+    # FIXME: https://github.com/actions/setup-python/issues/705
+    runs-on: ["Linux", "X64"] #${{ fromJSON(inputs.cloudformation_runs_on || inputs.runs_on) }}
```
Comment (klutchell) [inline flowzone.yml:3188]: This is forcing self-hosted runners, which are not available to all repositories, especially outside of our org.

Also it doesn't address the other jobs that are using setup-python.

**Comment by klutchell** [comment]
> > I prefer [this](https://github.com/product-os/flowzone/pull/953#issuecomment-2018787249) approach rather than hardcoding runner tags.
> 
> How is this approach fixing the underlying condition that the setup-python action will **only** currently succeed on X64/Linux? I must be missing something, perhaps you can add you commit directly to this branch?
> 
> We have seven environment repositories that are affected by this. We could technically change them all to only run on X64/Linux, but I think a one line change here is better than having to change x7, especially given that it will be temporary.

HOW temporary? Do we know if/when this will be fixed upstream?

I see the benefit of fixing it in Flowzone as it's less work, but then we have to make assumptions about what the calling repository has access to (self-hosted vs GitHub hosted) AND we need to remember and come back later to remove all the places in Flowzone where we are ignoring the `runs_on` input to provide a hardcoded one.

Would [this](https://github.com/actions/setup-python/issues/705#issuecomment-1756948951) be a suitable compromise?

**Comment by ab77** [comment]
> Would [this](https://github.com/actions/setup-python/issues/705#issuecomment-1756948951) be a suitable compromise?

Ugh..

File: flowzone.yml (hunk 1 above)
Comment (ab77) [reply flowzone.yml]: I am tempted to leave those to the project maintainer(s).

**Comment by ab77** [comment]
> > Would [this](https://github.com/actions/setup-python/issues/705#issuecomment-1756948951) be a suitable compromise?
> 
> Ugh..

We'll have to do it [upstream](https://github.com/balena-io/environment-playground/pull/62) and revert when they get their sh*t together.

**Review by klutchell — APPROVED** [review]

**Pull request merged**